recipes/pixstem/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pixstem" %}
+{% set version = "0.3.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 400da42fb11115b62548e8227f26b97c573870a72ebb5dbe66af437bec47c4a0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python =>3
+    - pip
+  run:
+    - python =>3
+    - scipy
+    - numpy >=1.13
+    - h5py
+    - matplotlib >=2.0
+    - dask
+    - tqdm
+    - scikit-image >=0.13
+    - scikit-learn
+    - hyperspy >=1.4
+
+test:
+  imports:
+    - pixstem
+  commands:
+    - export MPLBACKEND=agg  # [unix]
+    - set MPLBACKEND=agg  # [win]
+    - python -c "import pixstem.api"
+
+about:
+  home: https://pixstem.org
+  license: GPL-3.0
+  license_family: GPL
```

Rather than removing this field you should include a copy of the licence file in the recipe directory instead
```suggestion
  license_family: GPL
  license_file: LICENSE
```